src/bundle/Resources/public/scss/_filters.scss

```diff
@@ -34,4 +60,30 @@
     &__option--hidden {
         display: none;
     }
+
+    &__user-list {
+        padding: 0;
+        list-style: none;
+        position: absolute;
+        right: 0;
+        display: block;
```

Comment:
It can be omitted. Whenever you set `position: absolute` then the element gets `display: block` by default.